velox/connectors/hive/iceberg/tests/IcebergReadPositionalDeleteTest.cpp

```diff
@@ -587,196 +431,31 @@ class HiveIcebergTest : public HiveConnectorTestBase {
         numRowsInPreviousBaseFiles += baseFileSize.second;
       }
 
-      return fmt::format(
-          "SELECT * FROM tmp WHERE c0 NOT IN ({})",
-          makeNotInList(allDeleteValues));
-    }
-  }
-
-  std::vector<int64_t> flattenAndDedup(
-      const std::vector<std::vector<int64_t>>& deletePositionVectors,
-      int64_t baseFileSize) {
-    std::vector<int64_t> deletePositionVector;
-    for (auto vec : deletePositionVectors) {
-      for (auto pos : vec) {
-        if (pos >= 0 && pos < baseFileSize) {
-          deletePositionVector.push_back(pos);
+      std::string deleteValuesList;
+      if (!allDeleteValues.empty()) {
+        deleteValuesList = std::to_string(allDeleteValues[0]);
+        for (size_t i = 1; i < allDeleteValues.size(); ++i) {
```

Comment:
This is what `makeNotInList()` function is doing. Can we replace this with a call to that function? Also, `makeNotInList() `is not adding `NOT IN`, so we can change name to something like `getListAsCSVString()`